src/Auth/Login.tsx

```diff
@@ -0,0 +1,77 @@
+import { Form, Input, Button, Row, Col } from 'antd';
+import { EyeInvisibleOutlined, EyeTwoTone } from '@ant-design/icons';
+import { Link, useHistory } from 'react-router-dom';
+import { useAuth } from '../contexts/Auth';
+
+interface LoginProps {
+  email: string;
+  password: string;
+}
+export const Login = () => {
+  const [form] = Form.useForm();
+
+  // Get signUp function from the auth context
+  const { signIn } = useAuth();
+  const history = useHistory();
+
+  async function handleSumit(e: LoginProps) {
+
+    const email = e.email;
+    const password = e.password;
+
+    // Calls `signIn` function from the context
```

Comment:
You can remove these comments. Calling the function itself is enough to understand what the code is doing. Comments should be reserved for code that's hard to understand or to explain unorthodox decisions